### DISCUSSION TITLE
How to hide $q.notify notification after an async process has been completed

### DISCUSSION THREAD

**Discussion by eugenevk** [quasarframework/quasar#11928] [General - Components / Directives / etc] [answered]
I want to show a notification when a particular (asynchronous) process starts and hide it when the process has been completed. I only found the option to set a timeout, but that's not the solution. How can I determine when to hide the notification?

This is the current structure I have in a Vue component:
```
    const myFunction = async () => {
      $q.notify({
        color: 'secondary',
        message: 'Something is being done. Please wait...',
        position: 'bottom',
        spinner: QSpinnerGears,
        textColor: 'white'
      })

      await myProcess()

      // Now I want to hide the first notification

      $q.notify({
        color: 'green-8',
        icon: 'far fa-check-circle',
        message: 'Process has finished',
        position: 'bottom',
        textColor: 'white'
      })
    }
```

**Comment by rstoenescu** [comment, marked as answer]
https://quasar.dev/quasar-plugins/notify#updatable-notifications
or
https://quasar.dev/quasar-plugins/notify#programmatically-closing

**Reply by eugenevk** [reply]
Brilliant, solved! Thanks @rstoenescu
